### PR TITLE
[WIP] Adding _id label to cluster admin, cluster proxy, cluster proxy CA expiry Timestamp, cluster proxy ca valid and cluster id #93

### PR DESCRIPTION
**Pull request by anispate**
This PR is part of [OSD-14250](https://issues.redhat.com/browse/OSD-14250) ticket. The purpose of this PR is to add  _id label to cluster admin, cluster proxy, cluster proxy CA expiry Timestamp, cluster proxy ca valid and cluster id #93 #95

### CODE REVIEW

File: main.go
Comment (mjlshen) [inline main.go:128]: Make sure all the reconcilers you added the `ClusterId` field to get populated, e.g.

```suggestion
		MetricsAggregator: metrics.GetMetricsAggregator(clusterId),
		ClusterId:                  clusterId,
```

**Review by mjlshen — CHANGES_REQUESTED** [review]

**Comment by mjlshen** [comment]
/hold
Can be reviewed, but is more straightfoward after #92 to merges in first. Please feel free to unhold this whenever

**Comment by zmird-r** [comment]
Hi, I tested the operator and I confirm that the field` _id` is present for the `cluster_admin_enabled`, `cluster_proxy` and `cluster_id` metrics on my cluster.

```sh
[~ {stg} (zmird-qiu.oif6.s1:default)]$ curl --silent http://0.0.0.0:35000/metrics | grep cluster
# HELP cluster_admin_enabled Indicates if the cluster-admin role is enabled
# TYPE cluster_admin_enabled gauge
cluster_admin_enabled{_id="08e25af4-dede-4d59-b2df-7eeb819beb74",name="osd_exporter"} 0
# HELP cluster_id Indicates the cluster id
# TYPE cluster_id gauge
cluster_id{_id="08e25af4-dede-4d59-b2df-7eeb819beb74",name="osd_exporter"} 1
# HELP cluster_proxy Indicates cluster proxy state
# TYPE cluster_proxy gauge
cluster_proxy{_id="08e25af4-dede-4d59-b2df-7eeb819beb74",http="0",https="0",name="osd_exporter",trusted_ca="1"} 0
```

Thank you @T0MASD for the support.

**Pull request closed without merging**
